webcompat/api/endpoints.py

```diff
@@ -219,10 +220,9 @@ def proxy_comments(number):
     '''
     if request.method == 'POST':
         try:
-            comment_data = json.loads(request.data)
-            body = json.dumps({"body": comment_data['rawBody']})
             path = 'repos/{0}/{1}/comments'.format(ISSUES_PATH, number)
-            comment = github.raw_request('POST', path, data=body)
+            comment = github.raw_request('POST', path,
+                                         data=get_comment_data(request.data))
             return (json.dumps(comment.json()), comment.status_code,
                     {'content-type': JSON_MIME})
```

Comment:
Let's see for this full block here.  I was trying to understand what is happening and curious what the function would provide. It simplifies the route :+1: 

But I was wondering why the double dance dict -> json -> dict

Going to http://localhost:5000/issues/276 and posting a text comment with the TEST001 message.

`request.data`

``` python
{"avatarUrl":"https://avatars.githubusercontent.com/u/505230?v=3",
 "body":"<p>TEST001</p>\n",
 "commenter":"karlcow",
 "createdAt":"a few seconds ago",
 "commentLinkId":null,
 "rawBody":"TEST001"}
```

then  `get_comment_data(request.data)` is called.


```python
    comment_data = json.loads(request_data)
```

which is 

```python
{u'body': u'<p>TEST001</p>\n', 
 u'avatarUrl': u'https://avatars.githubusercontent.com/u/505230?v=3', 
 u'rawBody': u'TEST001', 
 u'commenter': u'karlcow', 
 u'commentLinkId': None, 
 u'createdAt': u'a few seconds ago'}
```

(funny how things changed orders)

Then `get_comment_data` returns

```python
 {"body": "TEST001"}
```

to `data` in github.raw_request which is creating a comment object and returns it.

`comment.json()` value is 

```python
{u'body': u'TEST001', 
 u'url': u'https://api.github.com/repos/webcompat/webcompat-tests/issues/comments/139690086', 
 u'created_at': u'2015-09-12T00:01:28Z', 
 u'html_url': u'https://github.com/webcompat/webcompat-tests/issues/276#issuecomment-139690086', 
 u'updated_at': u'2015-09-12T00:01:28Z', 
 u'user': {
    u'following_url': u'https://api.github.com/users/karlcow/following{/other_user}', 
    u'events_url': u'https://api.github.com/users/karlcow/events{/privacy}', 
    u'organizations_url': u'https://api.github.com/users/karlcow/orgs', 
    u'url': u'https://api.github.com/users/karlcow', 
    u'gists_url': u'https://api.github.com/users/karlcow/gists{/gist_id}', 
    u'html_url':u'https://github.com/karlcow', 
    u'subscriptions_url': u'https://api.github.com/users/karlcow/subscriptions', 
    u'avatar_url': u'https://avatars.githubusercontent.com/u/505230?v=3', 
    u'repos_url': u'https://api.github.com/users/karlcow/repos', 
    u'received_events_url': u'https://api.github.com/users/karlcow/received_events', 
    u'gravatar_id': u'', 
    u'starred_url': u'https://api.github.com/users/karlcow/starred{/owner}{/repo}', 
    u'site_admin': False, 
    u'login': u'karlcow', 
    u'type': u'User', 
    u'id': 505230, 
    u'followers_url': u'https://api.github.com/users/karlcow/followers'}, 
u'id': 139690086, 
u'issue_url': 
u'https://api.github.com/repos/webcompat/webcompat-tests/issues/276'}
```

And the dump reserializes it.